crates/floresta-common/src/lib.rs

```diff
@@ -18,11 +22,17 @@ pub mod spsc;
 use prelude::*;
 pub use spsc::Channel;
 
+/// Takes a reference to an array of bytes and returns
+/// a `sha256::Hash`.
 pub fn get_hash_from_u8(data: &[u8]) -> sha256::Hash {
     let hash = sha2::Sha256::new().chain_update(data).finalize();
     sha256::Hash::from_slice(hash.as_slice()).expect("Engines shouldn't be Err")
 }
 
+/// Takes a script and returns its hash following the Electrum protocol
+/// convention of reversing it.
+///
+/// The source to the specification can be found [here](https://electrum-protocol.readthedocs.io/en/latest/protocol-basics.html#script-hashes).
```

Comment:
```suggestion
/// Computes the sha256 of a script and reverses the bytes.
///
/// The source to the specification can be found in the Electrum protocol [documentation], and it is
/// used to identify scripts in the Electrum Protocol.
///
/// [documentation]: (https://electrum-protocol.readthedocs.io/en/latest/protocol-basics.html#script-hashes).
```

I think this way of doing links is way cleaner. I think we should add this to our coding standards.